BaseFilter/sections/specific.txt

```diff
@@ -1749,6 +1749,7 @@ champion.gg##.aside-content-column
 lolchess.gg##div[class^="ap_"].profile
 lolchess.gg##div[class*="Ad"]
 lolchess.gg##.ap_top
+totaljobs.com##article[data-testid="hp-app-banner-holder"]
```

Comment:
Please move this rule to - https://github.com/AdguardTeam/AdguardFilters/blob/master/AnnoyancesFilter/MobileApp/sections/mobile-app_specific.txt